src/components/alert/alert.html

```diff
@@ -61,41 +55,29 @@
     <div class="doc-content">
         <section class="component">
             <h1>Alert</h1>
-          <p class="md"> An alert is a dialog that presents users with information, alert messages. 
-            An alert appears on top of the web app's content and must be manually dismissed before user can resume
-            interacting with the app.
+          <p class="md"> An alert displays a short, important message in a way that attracts the user's attention without interrupting the user's task.
           </p>  
           </section>
-<h2>Demo</h2>
             <section class="demo">
-              <button id="call-alert" class="btn" onclick="showAlert()">Show Alert</button>
-          </section>
-          <div class="container alert" id="alert-container">
-            <div class="container-title">Alert Heading!</div>
-            <div class="container-content">Alert text description.</div>
-            <div class="container-footer">
-              <button class="btn btn-full-width" onclick="closeAlert()">Dismiss</button>
-            </div></div>
+            <div class="flex-col">
+            <div class="alert alert-success"> <span>This is a success alert</span> <span class="alert-close"> x </span></div>
```

Comment:
instead of adding **x** on the **alert icon button** add icons from icon-library like (material-icons, box icons)